### DISCUSSION TITLE
Multiple panel returning 404

### DISCUSSION THREAD

**Discussion by briantuju** [filamentphp/filament#13831] [Help]
### Package

Panel builder

### Package Version

v3.2.71

### How can we help you?

I have just created a second panel for my app using `php artisan make:filament-panel vendor`. This created a `VendorPanelProvider.php` file and it also updated the `providers.php` file by including the `VendorPanelProvider::class` in the array.

I also did `composer dump-autoload` and `php artisan optimize:clear`

I'm using Laravel **11.9**

Now when I try to go to `/vendor`, I get a 404 page like below:
![vendor-404](https://github.com/user-attachments/assets/0bf023fd-670d-452a-8e8a-4410c745fb74)

My `VendorPanelProvider` looks like this:
![vendor-panel-provider](https://github.com/user-attachments/assets/698531b9-c642-4786-b2ac-7f6179104f3c)

**Comment by briantuju** [comment]
I just created a new panel `vendors`, and it works as expected.

**Comment by skulls-dot-codes** [comment]
I had the same issues but it was caused by a `vendor` folder in `/public` which would give a `403` or `404` error. If you installed Laravel Telescope or another package then it might have created that folder for public assets. You can remove that folder if it's no longer needed since it doesn't get removed automatically.
